REASON Remove all the NERDTree-specific logic.

diff --git a/gvimrc b/gvimrc
index af0fe2c..2d85417 100644
--- a/gvimrc
+++ b/gvimrc
@@ -77,75 +77,6 @@ function StartTerm()
   setlocal listchars=tab:\ \ 
 endfunction
 
-" Project Tree
-if exists("loaded_nerd_tree")
-  autocmd VimEnter * call s:CdIfDirectory(expand("<amatch>"))
-  autocmd FocusGained * call s:UpdateNERDTree()
-  autocmd WinEnter * call s:CloseIfOnlyNerdTreeLeft()
-endif
-
-" Close all open buffers on entering a window if the only
-" buffer that's left is the NERDTree buffer
-function s:CloseIfOnlyNerdTreeLeft()
-  if exists("t:NERDTreeBufName")
-    if bufwinnr(t:NERDTreeBufName) != -1
-      if winnr("$") == 1
-        q
-      endif
-    endif
-  endif
-endfunction
-
-" If the parameter is a directory, cd into it
-function s:CdIfDirectory(directory)
-  let explicitDirectory = isdirectory(a:directory)
-  let directory = explicitDirectory || empty(a:directory)
-
-  if explicitDirectory
-    exe "cd " . fnameescape(a:directory)
-  endif
-
-  " Allows reading from stdin
-  " ex: git diff | mvim -R -
-  if strlen(a:directory) == 0 
-    return
-  endif
-
-  if directory
-    NERDTree
-    wincmd p
-    bd
-  endif
-
-  if explicitDirectory
-    wincmd p
-  endif
-endfunction
-
-" NERDTree utility function
-function s:UpdateNERDTree(...)
-  let stay = 0
-
-  if(exists("a:1"))
-    let stay = a:1
-  end
-
-  if exists("t:NERDTreeBufName")
-    let nr = bufwinnr(t:NERDTreeBufName)
-    if nr != -1
-      exe nr . "wincmd w"
-      exe substitute(mapcheck("R"), "<CR>", "", "")
-      if !stay
-        wincmd p
-      end
-    endif
-  endif
-
-  if exists(":CommandTFlush") == 2
-    CommandTFlush
-  endif
-endfunction
-
 " Utility functions to create file commands
 function s:CommandCabbr(abbreviation, expansion)
   execute 'cabbrev ' . a:abbreviation . ' <c-r>=getcmdpos() == 1 && getcmdtype() == ":" ? "' . a:expansion . '" : "' . a:abbreviation . '"<CR>'
@@ -166,21 +97,20 @@ function s:DefineCommand(name, destination)
   call s:CommandCabbr(a:name, a:destination)
 endfunction
 
-" Public NERDTree-aware versions of builtin functions
 function ChangeDirectory(dir, ...)
   execute "cd " . fnameescape(a:dir)
-  let stay = exists("a:1") ? a:1 : 1
-
-  NERDTree
 
-  if !stay
-    wincmd p
+  if exists(":CommandTFlush") == 2
+    CommandTFlush
   endif
 endfunction
 
 function Touch(file)
   execute "!touch " . shellescape(a:file, 1)
-  call s:UpdateNERDTree()
+
+  if exists(":CommandTFlush") == 2
+    CommandTFlush
+  endif
 endfunction
 
 function Remove(file)
@@ -193,19 +123,20 @@ function Remove(file)
     execute "!rm " . shellescape(a:file, 1)
   endif
 
-  call s:UpdateNERDTree()
+  if exists(":CommandTFlush") == 2
+    CommandTFlush
+  endif
 endfunction
 
 function Mkdir(file)
   execute "!mkdir " . shellescape(a:file, 1)
-  call s:UpdateNERDTree()
-endfunction
 
-function Edit(file)
-  if exists("b:NERDTreeRoot")
-    wincmd p
+  if exists(":CommandTFlush") == 2
+    CommandTFlush
   endif
+endfunction
 
+function Edit(file)
   execute "e " . fnameescape(a:file)
 
 ruby << RUBY
@@ -219,14 +150,11 @@ ruby << RUBY
 RUBY
 endfunction
 
-" Define the NERDTree-aware aliases
-if exists("loaded_nerd_tree")
-  call s:DefineCommand("cd", "ChangeDirectory")
-  call s:DefineCommand("touch", "Touch")
-  call s:DefineCommand("rm", "Remove")
-  call s:DefineCommand("e", "Edit")
-  call s:DefineCommand("mkdir", "Mkdir")
-endif
+call s:DefineCommand("cd", "ChangeDirectory")
+call s:DefineCommand("touch", "Touch")
+call s:DefineCommand("rm", "Remove")
+call s:DefineCommand("e", "Edit")
+call s:DefineCommand("mkdir", "Mkdir")
 
 " Include user's local vim config
 if filereadable(expand("~/.gvimrc.local"))

